docs/multicluster/user-guide.md

```diff
@@ -1,5 +1,20 @@
 # Antrea Multi-cluster User Guide
 
+## Table of Contents
+
+<!-- toc -->
+- [Quick Start](#quick-start)
+- [Installation](#installation)
+  - [Preparation](#preparation)
+  - [Deploy Antrea Multi-cluster Controller](#deploy-antrea-multi-cluster-controller)
+  - [Create ClusterSet](#create-clusterset)
+- [Multi-cluster Gateway Configuration](#multi-cluster-gateway-configuration)
+- [Multi-cluster Service](#multi-cluster-service)
+- [Multi-cluster ClusterNetworkPolicy Replication](#multi-cluster-clusternetworkpolicy-replication)
+- [Build Antrea Multi-cluster Image](#build-antrea-multi-cluster-image)
+- [Known Issue](#known-issue)
+<!-- /toc -->
```

Comment:
the file should be removed from `hack/.notableofcontents` to enable toc check

Reply:
Yes, I knew this. But I wanted not to add all levels of sub-titles to ToC, and so have to keep the file in `.notableofcontents` to skip the the ToC check.